### PR TITLE
Az 409 read txs from file

### PR DESCRIPTION
**Pull request by mike1729**

### CODE REVIEW

File: flooder/src/main.rs [hunk 1]
```diff
@@ -406,3 +432,16 @@ fn get_funds(connection: &Api<sr25519::Pair, WsRpcClient>, account: &AccountId)
         None => 0,
     }
 }
+
+fn zip_and_store_txs(txs: Vec<TransferTransaction> ) {
```
Comment (kostekIV) [inline flooder/src/main.rs:436]: some whitespace issues in this function

File: flooder/src/main.rs [hunk 2]
```diff
@@ -78,142 +79,167 @@ fn main() -> Result<(), anyhow::Error> {
...
+                .collect();
+
+                info!(
+                    "transactions signed: {}ms",
+                    time_stats.elapsed().as_millis()
+                );
+
+
+                if store_txs {
+                    zip_and_store_txs(txs);
```
Comment (kostekIV) [inline flooder/src/main.rs:215]: probably should be `zip_and_store_txs(txs, config.tx_store_path)`

File: flooder/src/main.rs (hunk 1 above)
Comment (mike1729) [reply flooder/src/main.rs]: Fixed.

File: flooder/src/main.rs (hunk 2 above)
Comment (mike1729) [reply flooder/src/main.rs]: Fixed.

**Review by kostekIV — APPROVED** [review]

**Pull request merged**